requirements.txt

```diff
@@ -0,0 +1,4 @@
+djangorestframework>=3.11.0,<3.12
+openwisp-utils>=0.4.4,<0.5.0
+pyasn1>=0.4.8,<0.5.0
+django-celery>=3.3.1,<3.4.0
```

Comment:
why are you adding this file here? It's redundant

Reply:
@nemesisdesign I am currently getting the following in my build logs after removing all unrequired lines. Also, I was previously getting the same traceback which led me to add them.
```
error: The 'djangorestframework' distribution was not found and is required by djangorestframework-gis
The command "python setup.py -q develop" failed and exited with 1 during 
```
& rest of the modules which I had added came up with similar error once this is resolved.
So can you please suggest edits to resolve this.